### DISCUSSION TITLE
Calling more options Charts

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#3602] [Q&A] [answered]
Hey,

is it possible to call more option when using a chart. 
I currently would need to call: **circumference** and **rotation**.

Trying to recreate this: https://jsfiddle.net/m846s85L/

Kind regards
Sebastian Gerner

**Comment by stsrki** [comment, marked as answer]
I think circumference and rotation are now part of the Data(DoughnutChartDataset). Starting from chartjs v3.
